controller/tests/models/domain/rsptag_test.py

```diff
@@ -359,3 +415,11 @@ def test_from_str() -> None:
     # them in dictionary form.
     for tag, expected in test_cases.items():
         assert asdict(RSPImageTag.from_str(tag)) == expected
+
+
+def test_age() -> None:
+    rsptag = RSPImageTag.from_str("d_2021_05_27_20210527")
```

Comment:
What's the extra stuff after the normal daily tag pattern?

Reply:
The tag I copied.  That'd be the "supplementary" part of the tag.  I presume we tossed those in to some of the tags for display name test purposes.  Dropping it will make the intent more clear.